app/views/assignments/_admin_subs.html.erb

```diff
@@ -132,14 +132,18 @@
   <table id="submissions" class="table table-striped row-vcenter">
     <thead>
       <tr>
-        <% if @gradesheet.assignment.team_subs? %>
-        <th>Team (submitter in bold)</th>
-        <% else %>
-        <th>Submitter</th>
-        <% end %>
+        <th>
+          <% if @gradesheet.assignment.team_subs? %>
+          Team (submitter in bold)
+          <% else %> 
+          Submitter
+          <% end %>
+          <button class="btn btn-default no-resort" id="show_username" title="Show/hide usernames" data-target="submissions">
```

Comment:
We probably want to turn this into a helper in app/helpers/application_helper.rb, 
```ruby
def toggle_usernames_button(id, target, class: nil)
  button_tag(type: nil, class: (class || "btn btn-default no-resort"), title: "Show/hide usernames", id: id, data: {target: target}) do
    content_tag :span, class: "glyphicon glyphicon-user", ""
  end
end
```
that can then be called as `<%= toggle_usernames_button("show_username", "submissions") %>`